cmd/common-methods.go

```diff
@@ -298,6 +298,11 @@ func uploadSourceToTargetURL(ctx context.Context, urls URLs, progress io.Reader,
 		}
 
 		sourcePath := filepath.ToSlash(sourceURL.Path)
+		if srcSSE != nil {
+			delete(metadata, "X-Amz-Server-Side-Encryption-Customer-Algorithm")
+			delete(metadata, "X-Amz-Server-Side-Encryption-Customer-Key-Md5")
```

Comment:
should also delete `x-amz-server-side-encryption` from metadata for SSE-S3 encrypted objects

Reply:
`srcSSE != nil` can be removed and all the source metadata with prefix `x-amz-server-side-encryption` should be removed. Because there can also be `x-amz-server-side-encryption-aws-kms-key-id`

Reply:
done